frontend/src/Components/Conditions/InfoButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import '../../css/InfoButton.css';
+
+/**
+ * A small button showing information and instructions to use the webapp.
+ */
+
+const InfoButton: React.FC = () => {
+  const [showInfo, setShowInfo] = useState(false);
+
+  const handleInfoToggle = () => {
+    setShowInfo(!showInfo);
+  };
+
+  return (
+    <>
+      <div className="info-button" onClick={handleInfoToggle}>
+        !
+      </div>
```

Comment:
I think this should be an 'i' instead of '!', but that's not v important

Reply:
<img width="193" alt="image" src="https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/45044551/d63ea288-98c7-49ec-a095-42221c7834f4">